src/questions/blockchain_basics.yaml

```diff
@@ -1,35 +1,1100 @@
+  choices:
+    - content: A Merger
+      key: A
+      order: 0
+    - content: A fork
+      key: B
+      order: 1
+    - content: A Division
+      key: C
+      order: 2
+    - content: None of the above
+      key: D
+      order: 3
+- uuid: 14443750-e46b-4c03-9c91-8434e55530dc
+  type: SingleChoice
+  content: Which of the following trees are used to store transactions in a block by producing a digital fingerprint?
+  hint: NoHint
+  explanation: Merkle trees are used to store transactions in a block by producing a digital fingerprint.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "High"
+  choices:
+    - content: Merkle
+      key: A
+      order: 0
+    - content: Binary
+      key: B
+      order: 1
+    - content: AVL
+      key: C
+      order: 2
+    - content: Red Black
+      key: D
+      order: 3
+- uuid: b76d8440-cf60-4069-9931-4855bc39c960
+  type: MultipleChoice
+  content: Who introduced the idea of Blockchain?
+  hint: NoHint
+  explanation: Stuart Haber and Scoot Storneta introduced blockchain to timestamp their documents.
+  answerKeys:
+    - A
+    - B
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Stuart Haber
+      key: A
+      order: 0
+    - content: Scoot Storneta
+      key: B
+      order: 1
+    - content: Satoshi Nakamoto
+      key: C
+      order: 2
+    - content: Adam Back
+      key: D
+      order: 3
+- uuid: 7e6aa6ff-d657-4d21-982e-fea86321c613
+  type: SingleChoice
+  content: Bitcoin is backed by which of the following assets?
+  hint: NoHint
+  explanation: Bitcoin does not have a tangible value and hence is not backed by any valueable asset.
+  answerKeys:
+    - D
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Gold
+      key: A
+      order: 0
+    - content: USD
+      key: B
+      order: 1
+    - content: Silver
+      key: C
+      order: 2
+    - content: None of the above
+      key: D
+      order: 3
+- uuid: 1c490831-d92a-449f-b5a2-08b453cc357f
+  type: SingleChoice
+  content: Who is the creator of Bitcoin?
+  hint: NoHint
+  explanation: Bitcoin was created by Satoshi Nakamoto.
+  answerKeys:
+    - D
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Adam Back
+      key: A
+      order: 0
+    - content: Cynthia Dwork
+      key: B
+      order: 1
+    - content: Moni Naor
+      key: C
+      order: 2
+    - content: Satoshi Nakamoto
+      key: D
+      order: 3
+- uuid: 3110ead9-fde0-43c0-8007-b1ba3a6781fd
+  type: SingleChoice
+  content: Like every digital asset Bitcoin has the problem of “double spend”?
+  hint: Double Spend means copying and reselling the same digital asset many number of times.
+  explanation: Every bitcoin has a unique hash pointer attached to it hence it becomes impossible to create a copy with a hash present in the system.
```

Comment:
Not sure if explanation is correct. The question also needs to be reworded.

It's an important question though https://www.investopedia.com/ask/answers/061915/how-does-block-chain-prevent-doublespending-bitcoins.asp

`The first transaction would be approved via the confirmation mechanism and then verified into the subsequent block. However, the second transaction would be recognized as invalid by the confirmation process and would not be verified. If both transactions are pulled from the pool for confirmation simultaneously, the transaction with the highest number of confirmations will be included in the blockchain, while the other one will be discarded.`

Reply:
Done